src/components/form/stepper/style.ts

```diff
@@ -0,0 +1,145 @@
+import { styled } from 'linaria/react';
+import { ThemeMode } from 'utils/types';
+
+export const StyledStepperInput = styled.div`
```

Comment:
```suggestion
export const StyledStepperInput = styled.div<{
mode: ThemeMode,
theme: Theme
}>`
```

you should use generics for this types of the components